crates/bevy_text/src/glyph_brush.rs

```diff
@@ -203,29 +207,47 @@ impl GlyphPlacementAdjuster {
 
 /// Computes the minimal bounding rectangle for a block of text.
 /// Ignores empty trailing lines.
-pub(crate) fn compute_text_bounds<'a, T>(
+pub(crate) fn compute_text_bounds<'a, T, I>(
     section_glyphs: &[SectionGlyph],
+    sections: I,
     get_scaled_font: impl Fn(usize) -> &'a PxScaleFont<T>,
 ) -> bevy_math::Rect
 where
+    I: DoubleEndedIterator<Item = &'a str> + ExactSizeIterator,
     T: ab_glyph::Font + 'a,
 {
-    let mut text_bounds = Rect {
-        min: Vec2::splat(std::f32::MAX),
-        max: Vec2::splat(std::f32::MIN),
-    };
-
-    for sg in section_glyphs {
-        let scaled_font = get_scaled_font(sg.section_index);
-        let glyph = &sg.glyph;
-        text_bounds = text_bounds.union(Rect {
-            min: Vec2::new(glyph.position.x, 0.),
-            max: Vec2::new(
-                glyph.position.x + scaled_font.h_advance(glyph.id),
-                glyph.position.y - scaled_font.descent(),
-            ),
-        });
+    let mut trailing_lines_space = 0.;
+    'outer: for (i, line) in sections.enumerate().rev() {
+        for ch in line.chars().rev() {
+            if ch == '\n' {
```

Comment:
should this be able to handle sequences like `\r\n\r\n`?

Reply:
https://doc.rust-lang.org/std/string/struct.String.html#method.lines is often a safer bet.

You could use it then iterate on each line